src/lib/autoinstall/ask/question.rb

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+module Y2Autoinstall
+  module Ask
+    # Represents a question related to an <ask> element
+    class Question
+      attr_accessor :text, :default, :help, :type, :password, :paths, :file,
+        :options, :element_id, :frametitle, :script, :default_value_script
+      attr_reader :stage, :value
+
+      # @!attribute text
+      #   @return [String] Question text
+
+      # @!attribute default
+      #   @return [Object,nil] Default value for the question
+
+      # @!attribute help
+      #   @return [String,nil] Help text
+
+      # @!attribute type
+      #   @return [String,nil] Question type ('symbol', 'boolean', 'string', 'integer'
+      #     or 'static_text')
+
+      # @!attribute password
+      #   @return [Boolean,nil] When set to `true`, it is supposed to be a password
+
+      # @!attribute paths
+      #   @return [Array<String>] Path of an element in the profile
```

Comment:
is it path of element or Paths? So how it can look like? and what means if there are multiple elements?